vertex-ai/managed_notebook/README.md

```diff
@@ -0,0 +1,61 @@
+## Vertex AI Workbench with Java Client Library
+
+This is a small sample to create a managed notebook using the Java SDK
+
+### Useful information
+
+Before compiling, open
+java-docs-samples/vertex-ai/managed_notebook/src/main/java/com/gcp_aiml/samples/CreateRuntime.java and update the parameters at the top of the `public class CreateRuntime`.  You will need to know these values:
+
+Project ID
+Project Number
```

Comment:
Why both?

Reply:
yeah, good point.  The code used to require them.  I fixed it in the code but forgot to update the docs.

Reply:
oh, I still see references to PROJECT_ID and PROJECT_NUMBER in the docs, for instance:

gcloud projects add-iam-policy-binding ${PROJECT_ID} \                                                                   
      --member="serviceAccount:${PROJECT_NUMBER}-compute@developer.gserviceaccount.com" \                                
      --role="roles/notebooks.admin"